lib/knackhq/client.rb

```diff
@@ -0,0 +1,42 @@
+require 'knackhq/client/version'
+require 'blanket'
+require 'hashie'
+
+module Knackhq
+  class Client
+    attr_accessor :base_uri, :x_knack_application_id, :x_knack_rest_api_key
+
+    def initialize(base_uri, x_knack_application_id, x_knack_rest_api_key)
+      @base_uri = base_uri
+      @x_knack_application_id = x_knack_application_id
+      @x_knack_rest_api_key = x_knack_rest_api_key
+    end
+
+    def objects
```

Comment:
Can you give me an example of your implementation?

I see the ClientQuery basically a means of layering method_missing onto of blanket but restricting the allowable queries down to Knack related names, object_x, etc.